src/middleware/validation.schema.js

```diff
@@ -82,6 +103,27 @@ const ratingSchema = {
   }
 };
 
+const articleSchema = {
+  body: {
+    title,
+    description,
+    content,
+    image
+  }
+};
+
+const updateArticleSchema = {
```

Comment:
You should have tests to show that these validations actually run. Provide bad data that don't conform to the schema and test that the validations stop the request.